impl/client/src/main/doc-js/pentaho/data/filter/spec/IIsGreater.jsdoc

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2017 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `spec.IIsGreater` interface describes the information associated with an equality filter.
+ *
+ * @name pentaho.data.filter.spec.IIsGreater
+ *
+ * @interface
+ * @extends pentaho.data.filter.spec.IProperty
+ *
+ * @see pentaho.data.filter.IsGreater
+ */
+
+/**
+ * The expected value of the property.
+ *
+ * @name value
+ * @memberOf pentaho.data.filter.spec.IIsGreater#
+ * @type pentaho.type.UValue
```

Comment:
`@type {pentaho.type.UValue}`. Applies to other filter types as well.
Assuming `null` can be specified, although the resulting filter will not be valid.